homework-submission/main.js

```diff
@@ -0,0 +1,67 @@
+const List = props => {
+  return (
+    <li>
+      {props.description},{props.deadline}
+    </li>
+  );
+};
+
+class TodoList extends React.Component {
+  staticList = [
+    { description: 'Get out of bed', deadline: 'Wed Sep 13 2017' },
+    { description: 'Brush teeth', deadline: 'Thu Sep 14 2017' },
+    { description: 'Eat breakfast', deadline: 'Fri Sep 15 2017' },
+  ];
+
+  dynamicList = [
+    {
+      id: 1,
+      description: 'Get out of bed',
+      deadline: '2017-09-11',
+      done: true,
+    },
+    {
+      id: 2,
+      description: 'Brush teeth',
+      deadline: '2017-09-10',
+      done: false,
+    },
+    {
+      id: 3,
+      description: 'Eat breakfast',
+      deadline: '2017-09-09',
+      done: false,
+    },
+  ];
+
+  render() {
+    return (
+      <div>
+        <h1>static List</h1>
+        <List description={this.staticList[0].description} deadline={this.staticList[0].deadline} />
+        <List description={this.staticList[1].description} deadline={this.staticList[1].deadline} />
```

Comment:
this comment is same as the previous, the `List` Component is returning an `li` element to a `div` parent.
making it confusing and invalid for the browser.

Please check the `Permitted parents` section of the [mdn docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/li) speaking of the `li` element